srcpkgs/i2pd/template

```diff
@@ -27,6 +27,13 @@ case "${XBPS_TARGET_MACHINE}" in
 esac
 
 do_install() {
```

Comment:
what about the shared library? you should be able to build that with an additional build target (`make_build_target="all api"` i think should work)

Reply:
ok, after looking at the makefile more, it seems they don't version the shared library, so there's not much point in using it, I think.